src/main/java/org/eclipse/cargotracker/interfaces/handling/rest/HandlingReport.java

```diff
@@ -8,35 +8,23 @@
 @XmlRootElement
 public class HandlingReport {
 
-    // TODO [Jakarta EE 8] Use not blank instead.
-    @NotNull(message = "Missing completion time.")
-    @Size(min = 16, max = 16, message = "Completion time value must be sixteen characters long.")
-    // TODO [DDD] Apply regular expression validation.
+    @NotNull
```

Comment:
This is a sync mismatch. Can you kindly restore the more updated code?

Reply:
update.